### ISSUE TITLE
Python 3.10 release

### ISSUE THREAD

**Issue by rt121212121** [coleifer/pysqlite3#29]
Is it possible to get a 3.10 release of pysqlite3-binary on Pypi? That would be a good help for getting some projects working with 3.10.

```
ERROR: Could not find a version that satisfies the requirement pysqlite3-binary (from versions: none)
ERROR: No matching distribution found for pysqlite3-binary
```

**Comment by goldsteinn** [comment]
You are missing `-DMODULE_NAME=\"sqlite3\"`

**Comment by rt121212121** [comment]
This has already been solved, the 3.10 wheels are available on PyPI. Thanks!
